.github/workflows/run_all_steps.yml

```diff
@@ -208,6 +208,7 @@ jobs:
             tmodel-ccs2018.github.io \
             --network_scale 0.0001 \
             --torperf_scale 0.1 \
+            --onion_service_traffic_scale 0.05 \
```

Comment:
If we implement my suggestion, this would either become 1.0 or could just be removed if we make 1.0 the default.